protocols/509b5d/Nucleic-Acid-Purification-HTP.ot2.py

```diff
@@ -0,0 +1,382 @@
+        pip300.aspirate(130, lysis)
+        pip300.dispense(130, well)
+        pip300.mix(3, 200, well)
+        pip300.blow_out(well.top())
+        pip300.return_tip()
+
+    # 10 minute incubation period
+
+    robot.comment('Incubating for 10 minutes. OT-2 will resume when incubation \
+    is complete.')
+    pip50.delay(minutes=10)
+    robot._driver.run_flag.wait()
+
+    # mix magbeads and dispense to each well
+
+    pip50.pick_up_tip(tips_1x.wells('A2'))
+
+    pip50.mix(5, 40, magbeads)
+
+    pip50.blow_out(magbeads.top())
+
+    pip50vol = 0
+
+    for well in samples50:
+        if pip50vol == 0:
+            pip50.aspirate(40, magbeads)
+            pip50vol += 40
+        pip50.dispense(20, well.top())
+        pip50vol -= 20
+
+    if pip50vol > 0:
+        pip50.dispense(pip50vol, pk)
+    pip50.drop_tip()
+
+    # add 500ul of binding solution to each well
+
+    pip300.pick_up_tip(tips_1x.wells('A3'))
+
+    for well in samples300:
+        pip300.transfer(250, binding, well.top(), new_tip='never')
+        pip300.blow_out(well.top())
+        pip300.transfer(250, binding, well.top(), new_tip='never')
+        pip300.blow_out(well.top())
+
+    pip300.drop_tip()
+
+    # re-use tips to mix and then replace tips in rack
+
+    for well in samples300:
+        well_name = well.get_name()
+        pip300.pick_up_tip(tips_reuse_a.wells(well_name))
+        pip300.mix(5, 280, well)
+        pip300.blow_out(well.top())
+        pip300.return_tip()
+
+    # incubate for 5 minutes
+    robot.comment('Incubating for 5 minutes. OT-2 will resume when incubation \
+    is complete.')
+    pip300.delay(minutes=5)
+    robot._driver.run_flag.wait()
+
+    # separate beads using magdeck
+
+    magdeck.engage(height=mag_offset)
+    pip300.delay(minutes=1)
+
+    # remove 870ul of supernatant from each well using replaced tips
+
+    for well in samples300:
+        well_name = well.get_name()
+        pip300.pick_up_tip(tips_reuse_a.wells(well_name))
+        for t in range(3):
+            pip300.transfer(290, well, waste, new_tip='never')
+            pip300.blow_out(waste.top())
+        pip300.return_tip()
+
+    magdeck.disengage()
+
+    # add 500ul of wash solution to each well with new tips
+
+    pip300.pick_up_tip(tips_1x.wells('A4'))
+
+    for well in samples300:
+        for t in range(2):
+            pip300.transfer(250, wash, well.top(), new_tip='never')
+            pip300.blow_out(well.top())
+
+    pip300.drop_tip()
+
+    # use tips from before to mix each well and then return tips
+
+    for well in samples300:
+        well_name = well.get_name()
+        pip300.pick_up_tip(tips_reuse_a.wells(well_name))
+        pip300.mix(5, 290, well)
+        pip300.blow_out(well.top())
+        pip300.return_tip()
+
+    # separate beads using magdeck
+
+    magdeck.engage(height=mag_offset)
+    pip300.delay(minutes=1)
+
+    # remove 500ul of supernatant with tips from before
+
+    for well in samples300:
```

Comment:
for tips, you can define the tips in a list at the beginning and zip that together with the samples